deploy/README.md

```diff
@@ -60,7 +60,7 @@ The following matrix displays the tested package versions for our Helm chart.
 
 Sumo Logic Helm Chart | Prometheus Operator | Fluent Bit | Falco  | Metrics Server
 |:-------- |:-------- |:-------- |:-------- |:--------
-0.14.0 - Latest | 8.2.0 | 2.8.1 | 1.1.1 | 2.7.0
+0.14.1 - Latest | 8.2.0 | 2.8.1 | 1.1.0 | 2.7.0
```

Comment:
we will have to add separate lines for `0.14.0` and `0.14.1`?

Reply:
I thought about it and I think we should not do it actually as `0.14.0` is not a version we'll support.
Let's just create matrix for the versions we support - and that is the `v0.14.1` with downgraded falco.
What do you think?

Reply:
hmm.. sounds good but since 0.14.0 is a relatively new release, I guess lot of people are using it. So, we first need to publish a note about just supporting the last two versions, and then we can remove it from the next release.

Reply:
+1, I believe many customers are probably using 0.14.0, even our infra team is using the 0.14.0 image (though their chart is different), and since the only change is the falco version I would leave a line for 0.14.0 for now since we still have rows for the previous versions as well

Reply:
👍 I've changed it to make it clear what versions are installed (for 0.14.0 - 0.16.0 and 0.17.0 - later)